mint/fakeAPIData.ts

```diff
@@ -0,0 +1,326 @@
+            title: 'years of intervention',
+            showline: true,
+            tickvals: ["prev_year_1", "prev_year_2", "prev_year_3"],
+            ticktext: [1, 2, 3],
+            autorange: true
+        },
+        yaxis: {
+            title: 'prevalence (%)',
+            showline: true,
+            range: [0, 20],
+            autorange: false
+        }
+    }
+};
+
+export const casesAvertedGraph =
+    {
+        data: [
+            {
+                x: ["ITN"],
+                cols: ["cases_averted"],
+                id: "ITN",
+                type: "bar",
+                name: "Pyrethoid ITN",
+                marker: {
+                    color: "blue",
+                    opacity: 0.5,
+                },
+                error_y: {
+                    type: "data",
+                    cols: ["cases_averted_high"],
+                    colsminus: ["cases_averted_low"],
+                    visible: true,
+                    thickness: 1.5,
+                    width: 0,
+                    opacity: 1
+                }
+            },
+            {
+                x: ["PBO"],
+                cols: ["cases_averted"],
+                id: "PBO",
+                type: "bar",
+                name: "Switch to Pyrethoid-PBO ITN",
+                marker: {
+                    color: "aquamarine",
+                    opacity: 0.5,
+                },
+                error_y: {
+                    type: "data",
+                    cols: ["cases_averted_high"],
+                    colsminus: ["cases_averted_low"],
+                    visible: true,
+                    thickness: 1.5,
+                    width: 0,
+                    opacity: 1
+                }
+            },
+            {
+                x: ["IRS"],
+                cols: ["cases_averted"],
+                id: "IRS",
+                name: "Only IRS",
+                type: "bar",
+                marker: {
+                    color: "purple",
+                    opacity: 0.5,
+                },
+                error_y: {
+                    type: "data",
+                    cols: ["cases_averted_high"],
+                    colsminus: ["cases_averted_low"],
+                    visible: true,
+                    thickness: 1.5,
+                    width: 0,
+                    opacity: 1
+                }
+            }
+        ],
+        layout: {
+            title: "Clinical cases averted per 1,000 people per year",
+            yaxis: {
+                title: "cases averted",
+                showline: true,
+                range: [0, 300],
+                autorange: false
+            },
+            xaxis: {
+                title: 'intervention',
+                showline: true,
+                tickvals: ["ITN", "PBO", "IRS"],
+                ticktext: ["ITN", "PBO", "IRS"],
+                autorange: true
+            },
+            showlegend: true
+        }
+    };
+
+export const settings = [
+    {
```

Comment:
I'm thinking these settings could use the dynamic form schema, aka the same as this: https://github.com/mrc-ide/hintr/blob/master/inst/schema/ModelRunOptions.schema.json

Reply:
Excellent idea. :+1: